### ISSUE TITLE
Attribute descriptions should be displayed as alt text in UI

### ISSUE THREAD

**Issue by jrtcppv** [cvisionai/tator#520]
UI inputs should display attribute type descriptions if they are available as alt text

**Comment by ermbutler** [comment]
https://github.com/cvisionai/tator/pull/532
